README.rst

```diff
@@ -1170,3 +1173,31 @@ be re-compiled in that case!
 
 .. _Makeit: https://git.llucax.com/w/software/makeit.git
 .. _fpm: https://github.com/jordansissel/fpm
+
+
+Coverage
+--------
+
+Compiling using code coverage can be done by passing ``COV=1`` to ``make``. If
+the D runtime supports the ``DRT_COV*`` environment variables (see list of
+version below), the coverage reports will be put in the ``$(COVDIR)`` directory
+(``$O/cov`` by default), otherwise they will be written in the top-level
+directory.
+
+Also by default the the coverage reports will be merged. To change this you can
```

Comment:
What does it mean that the reports will be merged?  Might be worth putting in an explanation.

Going by what follows I assume that this refers to what happens if one builds multiple times in a row without cleaning?